app/pages/processes.html

```diff
@@ -30,6 +30,9 @@
         td{
             padding: 0.5em;
         }
+        td:nth-child(n+6):nth-child(-n+9){
```

Comment:
**Update**: Suggest adding both **th** & **td** here for centering
`th:nth-child(n+6):nth-child(-n+9), td:nth-child(n+6):nth-child(-n+9){`